.circleci/config.pkl

```diff
@@ -8,7 +8,7 @@ prb {
 
 main = buildWorkflow
 
-local pklVersion = "0.25.1"
+local pklVersion = "0.25.2"
 local pklBinary = "https://repo1.maven.org/maven2/org/pkl-lang/pkl-cli-linux-amd64/\(pklVersion)/pkl-cli-linux-amd64-\(pklVersion).bin"
```

Comment:
In pkl-vscode we changed the download link to GitHub. We should do the same here.